Document max-db-size and auto-propagate in nscd.conf.

The code for nscd is available at:
http://sourceware.org/cgi-bin/cvsweb.cgi/libc/nscd/?cvsroot=glibc

The two new options are used in the default nscd.conf, introduced in
revisions 1.11 and 1.12.  See:
http://sourceware.org/cgi-bin/cvsweb.cgi/libc/nscd/nscd.conf.diff?r1=1.10&r2=1.12&cvsroot=glibc

See also the parser code for `nscd.conf', in `nscd/nscd_conf.c'.
To see how the max-db-size is used, grep `nscd/mem.c' for
`max_db_size'.
To se the actual usage of `auto-propagate', grep `nscd/pwdcache.c' and
`nscd/grpcache.c' for `propagate'.

A news entry where Ulrich Drepper announces the `auto-propagate'
option:
http://sourceware.org/ml/libc-alpha/2006-09/msg00065.html


diff -up man-pages-3.04/man5/nscd.conf.5.BAD man-pages-3.04/man5/nscd.conf.5
--- man-pages-3.04/man5/nscd.conf.5.BAD	2008-08-07 17:02:57.000000000 -0400
+++ man-pages-3.04/man5/nscd.conf.5	2008-08-07 17:04:02.000000000 -0400
@@ -183,6 +183,25 @@ is shared with the clients so
 that they can directly search in them instead of having to ask the
 daemon over the socket each time a lookup is performed.
 .RE
+
+.B max-db-size
+.I service
+.I number-of-bytes
+.RS
+The maximum allowable size for the service.
+.RE
+
+.B auto-propagate
+.I service
+.I <yes|no>
+.RS
+When set to
+.IR no " for " passwd " or " group " service, then the " .byname
+requests are not added to
+.IR passwd.byuid " or " group.bygid
+cache.
+This may help for tables containing multiple records for the same id.
+.RE
 .SH "SEE ALSO"
 .BR nscd (8)
 .\" .SH AUTHOR
